src/Pulsar.Client/Internal/ProducerImpl.fs

```diff
@@ -905,6 +905,10 @@ type internal ProducerImpl<'T> private (producerConfig: ProducerConfiguration, c
 
         member this.LastDisconnectedTimestamp =
             connectionHandler.LastDisconnectedTimestamp
+        member this.IsConnected =
+            match connectionHandler.ConnectionState with
+                | Ready _ -> true
+                | _ -> false
```

Comment:
Please ident `|` back to be right under `match`

Reply:
done